integration_tests/dbt_project.yml

```diff
@@ -1,24 +1,55 @@
 config-version: 2
 
 name: 'pendo_integration_tests'
-version: '1.0.0'
+version: '1.1.0'
 
 profile: 'integration_tests'
 
 dispatch:
   - macro_namespace: dbt_utils
     search_order: ['spark_utils', 'dbt_utils']
 
+models:
+  +schema: "pendo_{{ var('directed_schema','dev') }}"
+
 vars:
+  consistency_test_exclude:
+    ## String agg'd 
+    - rules
+    - page_rules
+    ## The previous logic can be unpredictable due to identical timestamps
+    - previous_feature_name
+    - previous_feature_page_id
+    - previous_feature_page_name
+    - previous_feature_product_area_name
+    - previous_feature_product_area_id
+    - previous_feature_id
+    - previous_feature_event_at
+    - previous_feature_num_minutes
+    - previous_feature_num_events
+    - previous_page_id
+    - previous_page_name
+    - previous_product_area_name
+    - previous_page_product_area_id
+    - previous_page_event_at
+    - previous_page_num_events
+    - previous_page_num_minutes
+    ## For union updates
+    - source_relation
+    - event_key
+    - feature_event_key
+    - page_event_key
+    - guide_event_key
+    - poll_event_key
```

Comment:
Let's remove these from release. We can include them locally when running the validation tests, but I wouldn't want these to be accidentally excluded in future validation runs.

Reply:
```suggestion
```